browser/ui/brave_rewards/rewards_panel_coordinator.cc

```diff
@@ -8,13 +8,15 @@
 #include <utility>
 
 #include "brave/components/constants/webui_url_constants.h"
-#include "chrome/browser/profiles/profile.h"
-#include "chrome/browser/ui/browser_window.h"
+#include "chrome/browser/ui/browser_window/public/browser_window_interface.h"
 
 namespace brave_rewards {
 
-RewardsPanelCoordinator::RewardsPanelCoordinator(Browser* browser)
-    : BrowserUserData<RewardsPanelCoordinator>(*browser) {}
+RewardsPanelCoordinator::RewardsPanelCoordinator(
+    BrowserWindowInterface* browser_window_interface)
+    : browser_window_interface_(browser_window_interface) {
+  DCHECK(browser_window_interface_);
```

Comment:
Nit: I think `CHECK` is preferred now unless an expensive calculation is involved.